lib/request-etag.js

```diff
@@ -62,7 +65,8 @@ function get(cache) {
 
 function Request (config) {
   var cache = new Cache(config);
-  return { get: get(cache) };
+  request.get = get(cache);

```

Comment:
By the way.. this was my initial approach.. but it would have introduced two problems (I think..)
* `var x = new Request(conf1);` `var y = new Request(conf2);` would actually be the same instance, so we would have `x === y
* this one I need to check.. but depending on how the require cache works, we may be modifying the request module for the whole app using the request-etag module. This is definitely not something that we want to do. (but again, I am not sure about this one..)